agent/php_environment.c

```diff
@@ -382,6 +382,61 @@ static void nr_php_gather_dispatcher_information(nrobj_t* env) {
 
   nro_set_hash_string(env, "Dispatcher", dstring);
 }
+void nr_php_process_environment_variable(const char* prefix,
```

Comment:
Could this be declared static or is it something that would be used by other source files?